dotCMS/src/main/java/com/dotcms/publisher/pusher/PushPublisher.java

```diff
@@ -532,10 +532,7 @@ private String getFormattedPort(String port){
 	 * @return The REST {@link Client}.
 	 */
 	private Client getRestClient() {
-		if (null == this.restClient) {
```

Comment:
did you remove the attribute on the class referrer by this.restClient?

Reply:
done https://github.com/dotCMS/core/pull/20385/commits/d11e66e10755f5112d5c8cdb341fe59643b84eca